Gemfile

```diff
@@ -125,6 +125,7 @@ end
 group :test do
   gem 'capybara', '>= 3.32.2', '< 4.0'
   gem 'climate_control' # Allows environment variables to be modified within specs
+  gem 'cucumber', '~> 3.0', require: false
   gem 'cucumber-rails', '>= 2.0.0', require: false
```

Comment:
Looks good, just one thing.

cucumber-rails **isn't** being updated. 
cucumber **is being** locked at 3 instead.

So the title is misleading but since there's dependencies, is that alright? Nothing blocking tbh.

Reply:
Cucumber-rails is being updated in the Gemfile.lock by dependabot
Part of that automatically updated Cucumber because we didn't specify a specific version.
From now on only a manual update (or a breaking security fix) will push the major version of Cucumber above 3

Reply:
Ignore that, sorry just saw that it's updating it's cucumber-rails